mopidy/core/library.py

```diff
@@ -273,6 +296,12 @@ def search(self, query=None, uris=None, exact=False, **kwargs):
                 logger.warning(
                     '%s does not implement library.search() with "exact" '
                     'support. Please upgrade it.', backend_name)
+            except LookupError:
+                raise
```

Comment:
Please add a comment, and maybe a TODO about stopping the use of exceptions in the core API?